profiles/base.yaml

```diff
@@ -269,7 +269,13 @@ steal_past_mindlock: false
 stealing_buffs:
 npc_stealing_attempt_count: 0
 dont_steal_list:
-
+stealing_towns:
```

Comment:
Do these default stealing_towns match how stealing is currently behaving?

Reply:
Yes, these are exactly the towns where items are already in the db.